src/Okta.Sdk/Generated/ApplicationVisibility.Generated.cs

```diff
@@ -16,13 +16,6 @@ namespace Okta.Sdk
     /// <inheritdoc/>
     public sealed partial class ApplicationVisibility : Resource, IApplicationVisibility
     {
-        /// <inheritdoc/>
-        public bool? AutoLaunch 
```

Comment:
Are we expecting this change? Just curious why this property was added in the previous PR but now we're removing it.

Reply:
Yes! I noticed that too. v2.6.0 spec has changed between those PRs